docs/operators/how-to/configure-store.md

```diff
@@ -1,56 +1,57 @@
+## Configuring the node as a waku store client
 
-Ensure that `store` is enabled (this is `true` by default) and provide at least one store service node address with the `--storenode` CLI option.
+Provide at least one store service node address with the `--storenode` option. This option is independent of the `--store` option i.e., one node can act as a waku store client without mounting the Waku Store protocol.
 
-See the following example, using the peer at `/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/30303/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm` as store service node.
+For example, to use the peer at `/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/30303/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm` as the waku store service node:
 
-```sh
-./build/wakunode2 \
-  --store:true \
-  --storenode:/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/30303/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm
+```shell
+wakunode2 \
+  --storenode=/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/30303/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm
 ```
 
 Your node can now send queries to retrieve historical messages
-from the configured store service node.
-One way to trigger such queries is asking your node for historical messages using the [Waku v2 JSON RPC API](https://rfc.vac.dev/spec/16/).
-
-## Configuring a store service node
-
-To store historical messages on your node which can be served to store clients the `--persist-messages` CLI option must be enabled.
-By default a node would store up to the latest `50 000` messages.
-This is configurable using the `--store-capacity` option.
-A node that has a `--db-path` set will backup historical messages to a local database at the DB path
-and persist these messages even after a restart.
-
-```sh
-./build/wakunode2 \
-  --store:true \
-  --persist-messages:true \
-  --db-path:/mnt/nwaku/data/db1/ \
-  --store-capacity:150000
+from the configured store service node. One way to trigger such queries is asking your node for historical messages using the [Waku v2 JSON RPC API](https://rfc.vac.dev/spec/16/).
+
+## Configuring the node as a store service node
+
+If the waku store node is enabled (the `--store` option is set to `true`) the node will store historical messages and will be able to serve those messages to the waku store clients.
+
+There is a set of configuration options to customize the waku store protocol's message store. These are the most relevant:
+
+* `--store-message-retention-policy`: This option controls the retention policy i.e., how long certain messages will be persisted. Two different retention policies are supported:
+  + The time retention policy,`time:<duration-in-seconds>` (e.g., `time:14400`)
+  + The capacity retention policy,`capacity:<messages-count>` (e.g, `capacity:25000`)
+  + To disable the retention policy, explicitly, set this option to to `""`, an empty string.
```

Comment:
Can we spell out the implications? Something like: `Disabling the retention policy implies that, from nwaku's perspective, messages will either be persisted indefinitely or that retention is managed by a third party outside of the node.`